### ISSUE TITLE
extra line in notcurses-info when any scrolling is performed

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#2021]
When any scrolling is performed by `notcurses-info` (i.e. when it is started within 15 rows of the bottom), we get an extra blank line between the bottom line of stdout (the emoji row) and the first line of stderr (the render stats). This doesn't happen if we don't scroll. It's not a big deal, but it probably represents a bigger underlying problem.

**Comment by dankamongmen** [comment]
I think this was just resolved in acbfa8dab. We were resetting `rstate.{x,y}` in `notcurses_core_init()` well after we ought have. I've removed this duplicate, and we no longer see the extra line. This had the following effect at startup:

```
going to 69/0 from -1/-1                                                                                                  
NEW Y/X: 69/0                                                                                                             
going to 70/0 from -1/-1                                                                                                  
NEW Y/X: 70/0
```

leading to an extra scroll event.

**Comment by dankamongmen** [comment]
we managed to reintroduce this while fixing up #2038, fml
